packages/design-system/src/components/UsaBanner/I18nUsaBanner.jsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable filenames/match-exported */
+import { I18nextProvider } from 'react-i18next';
+import PropTypes from 'prop-types';
+import React from 'react';
+import UsaBanner from './UsaBanner';
+import i18n from '../i18n';
+
+/**
+ * A container component responsible for passing an instance
+ * of i18next to all child components using react-i18next's
+ * `translate` HOC. Note that we use I18nextProvider in order
+ * to avoid conflicts with other apps using react-i18next.
+ * See https://github.com/i18next/react-i18next/issues/382 for
+ * more context on why we need to do it this way.
+ */
+const I18nUsaBanner = function (props) {
+  return (
+    <I18nextProvider i18n={i18n} initialLanguage={props.initialLanguage}>
```

Comment:
Instead of making a separate wrapper file, can we just use `I18nextProvider` inside `UsaBanner`?

Reply:
That's a good suggestion, will look into it.